docs/source/cupy-reference/environment.rst

```diff
@@ -0,0 +1,22 @@
+Environment variables
+=====================
+
+Here are environment variables Chainer uses.
+
+
++--------------------+----------------------------------------------------+
+| ``CUPY_CACHE_DIR`` | Path to the directory to store kernel cache.       |
+|                    | ``$(HOME)/.cupy.kernel_cache`` is used by default. |
+|                    | See :ref:`cupy-overview` for detail.               |
++--------------------+----------------------------------------------------+
+
+
+For install
+-----------
+
+These environment variables are only referred on intall.
+
++---------------+----------------------------------------+
+| ``CUDA_PATH`` | Path to the directory containing CUDA. |
```

Comment:
The default would be the directory where `nvcc` is, right? Could you please note that.